docker/.env.dist

```
@@ -30,10 +30,10 @@ CANARY_TEST_ACCOUNTS=true
 CANARY_DATA_PACK=data-otservbr-global
 CANARY_MAP_URL=https://github.com/opentibiabr/canary/releases/download/v3.6.1/otservbr.otbm
 
-# MyAAC site. The Dockerfile builds from the MyAAC develop branch by default.
+# MyAAC site. The Dockerfile builds from the MyAAC 2.x branch by default.
 MYAAC_HTTP_PORT=8080
 MYAAC_SITE_URL=http://localhost:8080
-MYAAC_REF=develop
+MYAAC_REF=2.x
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MYAAC_REF key should go before the MYAAC_SITE_URL key

(UnorderedKey)